nevergrad/benchmark/plotting.py

```diff
@@ -233,16 +234,16 @@ def create_plots(
             try:
                 if name == "fight_all.png":
                     with open(str(output_folder / name) + ".cp.txt", "w") as f:
-                        f.write("ranking:\n")
-                        for i, algo in enumerate(data_df.columns[:8]):
-                            f.write(f"  algo {i}: {algo}\n")
+                       f.write("ranking:\n")
+                       for i, algo in enumerate(data_df.columns[:8]):
+                           f.write(f"  algo {i}: {algo}\n")
             except:
                 pass
             if len(name) > 80:
-                hash_ = hashlib.md5(bytes(name, 'utf8')).hexdigest()
-                name = re.sub(r'\([^()]*\)', '', name)
-                mid = len(name) // 2
-                name = name[:mid] + hash_ + name[mid:]
+                hash = hashlib.md5(bytes(name, 'utf8')).hexdigest()
```

Comment:
dont override hash, which is a reserved name

Reply:
Done, thx.